packages/h5p-server/src/ContentUserDataManager.ts

```diff
@@ -0,0 +1,154 @@
+import {
+    ContentId,
+    IContentUserData,
+    IUser,
+    IContentUserDataStorage
+} from './types';
+
+import Logger from './helpers/Logger';
+
+const log = new Logger('ContentUserDataManager');
+
+/**
+ * The ContentUserDataManager takes care of saving user data and states. It only contains storage-agnostic functionality and
+ * depends on a ContentUserDataStorage object to do the actual persistence.
+ */
+export default class ContentUserDataManager {
+    /**
+     * @param contentUserDataStorage The storage object
+     */
+    constructor(contentUserDataStorage: IContentUserDataStorage) {
```

Comment:
Using a private constructor argument eliminated the need to assign the property and even declare it.